html/pfappserver/root/static.alt/src/globals/configuration/pfConfigurationSyslogParsers.js

```diff
@@ -0,0 +1,451 @@
+            }
+          ]
+        },
+        {
+          label: i18n.t('Enabled'),
+          fields: [
+            {
+              key: 'status',
+              component: pfFormRangeToggle,
+              attrs: {
+                values: { checked: 'enabled', unchecked: 'disabled' }
+              }
+            }
+          ]
+        },
+        {
+          label: i18n.t('Alert pipe'),
+          fields: [
+            {
+              key: 'path',
+              component: pfFormInput,
+              validators: {
+                [i18n.t('Value required.')]: required,
+                [i18n.t('Maximum 255 characters.')]: maxLength(255)
+              }
+            }
+          ]
+        },
+        {
+          if: ['regex'].includes(syslogParserType), // 'regex' only
+          label: 'Rules',
+          fields: [
+            {
+              key: 'rules',
+              component: pfFormFields,
+              attrs: {
+                buttonLabel: 'Add Rule - New ( )',
+                sortable: true,
+                field: {
+                  component: pfFieldRuleSyslogParserRegex,
+                  attrs: {
+                    actions: {
+                      component: pfFieldApiMethodParameters,
+                      attrs: {
+                        typeLabel: i18n.t('Select action type'),
+                        valueLabel: i18n.t('Select action value'),
+                        fields: [
+                          pfConfigurationSyslogParserRegexRuleActions.deregister_node_ip,
+                          pfConfigurationSyslogParserRegexRuleActions.role_detail,
+                          pfConfigurationSyslogParserRegexRuleActions.modify_person,
+                          pfConfigurationSyslogParserRegexRuleActions.register_node_ip,
+                          pfConfigurationSyslogParserRegexRuleActions.add_person,
+                          pfConfigurationSyslogParserRegexRuleActions.update_ip6log,
+                          pfConfigurationSyslogParserRegexRuleActions.unreg_node_for_pid,
+                          pfConfigurationSyslogParserRegexRuleActions.trigger_scan,
+                          pfConfigurationSyslogParserRegexRuleActions.reevaluate_access,
+                          pfConfigurationSyslogParserRegexRuleActions.update_ip4log,
+                          pfConfigurationSyslogParserRegexRuleActions.update_role_configuration,
+                          pfConfigurationSyslogParserRegexRuleActions.trigger_security_event,
+                          pfConfigurationSyslogParserRegexRuleActions.release_all_security_events,
+                          pfConfigurationSyslogParserRegexRuleActions.modify_node,
+                          pfConfigurationSyslogParserRegexRuleActions.close_security_event,
+                          pfConfigurationSyslogParserRegexRuleActions.register_node,
+                          pfConfigurationSyslogParserRegexRuleActions.dynamic_register_node
+                        ]
```

Comment:
Could we just use `Object.values(pfConfigurationSyslogParserRegexRuleActions)` instead?

Reply:
I prefer this - although verbose - as it allows alphabetical ordering in the definition, and static sorting in the reference. If we combine as you suggest we'll lose one of these benefits.